### DISCUSSION TITLE
Support regex repo pattern

### DISCUSSION THREAD

**Discussion by jpoehnelt** [BetaHuhn/repo-file-sync-action#127] [Ideas]
We have many repositories in our org. It would be nice to match on some prefix or other pattern.

```yml
repos: |
  org/foo-.*$
  org/foobar
```

Constraint would require always having a defined org.

**Comment by BetaHuhn** [comment]
Related to [#83 (Branch Name Wildcard Support)](https://github.com/BetaHuhn/repo-file-sync-action/discussions/83). I think it makes sense to support regex in both branches and the repository name.

This is definitely something I want to add, I just don't have the time at the moment to work on this myself.

If you or anyone else wants to create a PR, happy to review and merge it.
